runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -239,41 +265,11 @@ public TxnContext txn(@Nonnull final String namespace, IsolationLevel isolationL
     public ScopedTransaction scopedTxn(
             @Nonnull final String namespace, IsolationLevel isolationLevel,
             Table<?, ?, ?>... tables) {
-        try {
-            return IRetry.build(IntervalRetry.class, () -> {
```

Comment:
Hi @vjeko - apologies if I missed this, but where is this retry moved to? I don't see it in the cnostructor around txBeginInternal right? The MVCC layer can throw a `TrimmedException` which we need to catch and retry when getting a scoped transaction snapshot right?
This was actually an issue that we found during our scale demo testing